examples/verify.rs

```diff
@@ -46,11 +47,14 @@ macro_rules! measure {
 fn run(num_messages: usize) {
     println!("dancing with {} messages", num_messages);
 
-    let rng = &mut XorShiftRng::from_seed([0x3dbe6259, 0x8d313d76, 0x3237db17, 0xe5bc0654]);
+    let mut rng = XorShiftRng::from_seed([
+        0x59, 0x62, 0xbe, 0x5d, 0x76, 0x3d, 0x31, 0x8d, 0x17, 0xdb, 0x37, 0x32, 0x54, 0x06, 0xbc,
```

Comment:
Is it intentional that the forth byte changed from 0x3d to 0x5d?

Reply:
just adding some bytes, not particularly important as it is just the test seed